environments/jvm/Dockerfile-11

```diff
@@ -0,0 +1,16 @@
+FROM maven:3.6-jdk-11 as BUILD
+WORKDIR /usr/src/myapp/
+
+# To reuse the build cache, here we split maven dependency
+# download and package into two RUN commands to avoid cache invalidation.
+COPY pom.xml .
+RUN mvn dependency:go-offline
+
+COPY src /usr/src/myapp/src/
+RUN mvn package
+
+FROM openjdk:11-jre
+VOLUME /tmp
+COPY --from=BUILD /usr/src/myapp/target/env-java-0.0.1-SNAPSHOT.jar /app.jar
+ENTRYPOINT java ${JVM_OPTS} -Djava.security.egd=file:/dev/./urandom -jar /app.jar --server.port=8888
```

Comment:
What does the java.security.egd do?

Reply:
The argument `-Djava.security.egd=file:/dev/./urandom` tells the JVM to run use` /dev/urandom` instead of `/dev/random` for strong random numbers. This is done to workaround the problem of less entropy on docker containers, which reportedly can be a problem for JVM blocking or the boot speed.